### PR TITLE
Deprecated 'k8s_minikube_vm_driver' parameter in session.

### LINKED ISSUE #54: Remove all `show_logs=True` parameter when initializing a session.[BUG]
**Describe the bug**

The `show_log=True` parameter has been removed in #42 but there are still many occurrences in our codebase.

**Expected behavior**

Don't use removed parameter.

### PR DESCRIPTION
**Pull request by lidongze0629**
## What do these changes do?

Minikube can not be deployed with non-driver in mac OS,  so we deprecate the k8s_minikube_vm_driver in session which is a bit hacky.
This PR also fixes #54 #106 

## Related issue number

Fixes #54 #106 #114

### CODE REVIEW

File: python/tests/test_session.py [hunk 1]
```diff
@@ -29,6 +29,8 @@
 import graphscope
 from graphscope.client.session import DEFAULT_CONFIG_FILE
 
+graphscope.set_option(show_log=True)
```
Comment (sighingnow) [inline python/tests/test_session.py:32]: Put the global setting to `setUpModule`.

c.f.: https://docs.python.org/3/library/unittest.html#setupmodule-and-teardownmodule

**Review by sighingnow — APPROVED** [review]
I have noticed comment from Weibin in #114:

> the 'kind' is not very OK for Mac, it need some hack tool like [docker-tuntap-osx](https://github.com/AlmirKadric-Published/docker-tuntap-osx) and [MetalLB](https://metallb.universe.tf/) to Load Balancing on Mac.
> 
> I think the Kubernetes of Docker for Mac with Load Balancing is enough to getting started on Mac.
> 
> ref: https://www.thehumblelab.com/kind-and-metallb-on-mac/#_

Could you please drop all word "Minikube" in documentation (as well as README.md)? It there's any...

**Comment by acezen** [comment]
Do we need to replace minikube with KinD since the prepare_env.sh use KinD on Linux/WSL2?

**Comment by sighingnow** [comment]
> Do we need to replace minikube with KinD since the prepare_env.sh use KinD on Linux/WSL2?

Continuing to use minikube in CI is fine for me, that would be helpful for avoiding further debugging work for CI. The usage of minikube is transparent, though. (and won't affect end users)

**Pull request merged**